gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
this one is autogenerated. let's not change autogenerated stuff

Reply:
yes, it was re-generated by wrapper task

Reply:
https://github.com/gradle/gradle/commit/58e5c9f696d74127d42540e00514881e7491fc9b#diff-3ce07ba0083d748d3a1d38fb9a602643